assets/css/_sass/website/_home.scss

```diff
@@ -11,15 +11,15 @@
     flex: 1;
   };
   h2 {
-    font-size: $font-size-5;
+    font-size: $font-size-5 !important;
     font-weight: $regular-weight;
     color: $grey400;
     margin: 40px auto 0px auto;
   };
   p {
     font-size: $font-size-3;
     font-weight: $regular-weight;
-    color: $grey600 !important;
+    color: $grey400 !important;
```

Comment:
Ty's design had the body copy in the footer at `$grey600`. Are we changing that?

Reply:
Hmmm... this is a good question. The JIRA card says to change the footer copy to 400. After looking at the annotated designs it appears that both gray400 and gray600 are used. I think the gray600 should only apply to the "Sign up for updates..." portion of the footer. I'll dig into the source file and adjust accordingly.

Reply:
Ok I just checked in Ai and this change is correct. Both instances of body copy in the footer on this page are gray400 so this change can stay.

Reply:
Change staying per discussion ^